### PR TITLE
[Keyboard] Fix function references IDB 60

### PR DESCRIPTION
**Pull request by drashna**
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* compiler issue on gcc 8+

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/idb/idb_60/idb_60.h [hunk 1]
```diff
@@ -2,24 +2,13 @@
...
-    writePinLow(C4);
-}
+void _idb_60_caps_led_on(void);
+void _idb_60_fn_led_on(void);
+void _idb_60_caps_led_off(void);
+void _idb_60_fn_led_off(void);
 
-#define _idb_60_esc_led_on _idb_60_fn_led_on
-#define _idb_60_esc_led_off _idb_60_fn_led_off
+#define _idb_60_esc_led_on()_idb_60_fn_led_on()
```
Comment (fauxpark) [inline keyboards/idb/idb_60/idb_60.h:10]:
```suggestion
#define _idb_60_esc_led_on() _idb_60_fn_led_on()
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
@itspngu Just a heads up on the change

**Review by fauxpark — APPROVED** [review]

**Review by itspngu — CHANGES_REQUESTED** [review]
Hey! Thanks, I actually meant to look into the reports of this not compiling anymore, but never got around to it. Since you're already on this, apparently these got mixed up somewhere down the line, so the LEDs are driven in inverse (they're on when they're meant to be off and vice versa).

**Comment by drashna** [comment]
Not a problem, at all!  Mostly wanted to make sure it compiled, but more than willing to add some other fixes, too!

**Comment by itspngu** [comment]
> 
> 
> Not a problem, at all! Mostly wanted to make sure it compiled, but more than willing to add some other fixes, too!

Awesome, thanks!

**Pull request merged**